engine/src/hopeit/toolkit/auth.py

```diff
@@ -141,14 +141,14 @@ def app_private_key(app_key: str) -> RSAPrivateKey:
 
 def new_token(app_key: str, payload: dict) -> str:
     private_key = app_private_key(app_key)
-    token = jwt.encode(
+    token = str(jwt.encode(
```

Comment:
if jwt.encode returns bytes the right way to convert to string is using decode()

Reply:
It returns str in the new version, the str casting is not needed.-